### DISCUSSION TITLE
Using Tauri app on Blink Build

### DISCUSSION THREAD

**Discussion by danilopezmella** [blinksh/blink#1832] [Support]
Hi guys. 

I'm starting in the world of developing apps and I would love to use my ipad to build a Tauri apps.

https://tauri.app

Is there a workaround to make develop an app in blink build?

**Comment by carloscabanero** [comment]
Hi! I never heard of Tauri until now, but I don't see why not. Just install their framework on Blink Build and should be good to go. What Im not sure is how you would then install on the device, but looks like they have a self updater, etc... so they may already have an app you can use to bootstrap.

Try it out and let us know if you have any questions along the way. It would also make a great QuickStart guide for our docs!
